appscale/tools/templates/AppScalefile-cloud

```diff
@@ -60,6 +76,18 @@ max : 1
 # Compute Engine on your behalf.
 #client_secrets: '~/client_secrets.json'
 
+# The Azure subscription ID for your account to use for AppScale deployment.
+#subscription_id :
```

Comment:
Please add the equivalent of 'project-id-here' below for each of this parameters.